### PR TITLE
Won't display verbal feedback if there are no poses detected

### PR DESCRIPTION
**Pull request by n63li**
Screen Shot 2021-03-11 at 1 11 07 PM

### CODE REVIEW

File: VirtualTrainer.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -166,7 +166,6 @@
 				D8C6932225E8A0E10012C5F6 /* WorkoutResultViewController.swift */,
 				914F873625F9476F005BB3DE /* OverlayVideoView.swift */,
 				914F86F125F4690F005BB3DE /* TrendsViewController.swift */,
-				FBACAE2325EFFCE20000174A /* IMUPeripheral.swift */,
```
Comment (henrymxu) [inline VirtualTrainer.xcodeproj/project.pbxproj:169]: i dont think you can remove this?

Comment (henrymxu) [reply VirtualTrainer.xcodeproj/project.pbxproj:169]: nvm, seems like its duplicated

File: VirtualTrainer/FeedbackViewController.swift [hunk 2]
```diff
@@ -28,37 +28,43 @@ class FeedbackViewController: UIViewController {
...
+                scoreLabel?.text = "Could not detect proper postures - are you sure you uploaded the correct video for the selected exercise and camera angle?"
+            }
         }
-
+        else {
+            let feedback = workoutSession?.generateFeedback()
+            var feedbackParagraph = feedback?[0]
+            
+            for sentence in feedback![1...] {
+                feedbackParagraph! += " " + sentence
```
Comment (henrymxu) [inline VirtualTrainer/FeedbackViewController.swift:62]: add new line

Comment (imptrx) [inline VirtualTrainer/FeedbackViewController.swift:62]: Newline here

File: VirtualTrainer/WorkoutResultViewController.swift [hunk 3]
```diff
@@ -21,26 +21,31 @@ class WorkoutResultViewController: UIViewController {
...
+            scoreLabel?.text = "Could not detect proper postures - are you sure you uploaded the correct video for the selected exercise and camera angle?"
+            feedbackLabel.isHidden = true
+        }
+        else {
+            scoreLabel?.text = "You have achieved a score of \(workoutSession!.workoutResult.score!)"
+            let feedback = workoutSession?.generateFeedback()
+            var feedbackParagraph = feedback?[0]
+            
+            for sentence in feedback![1...] {
+                feedbackParagraph! += " " + sentence
```
Comment (imptrx) [inline VirtualTrainer/WorkoutResultViewController.swift:34]: .

**Review by henrymxu — APPROVED** [review]

**Review by jeremyNCH — APPROVED** [review]

**Pull request merged**